protected/humhub/modules/admin/jobs/CleanupPendingRegistrations.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\admin\jobs;
+
+use humhub\modules\queue\ActiveJob;
+use humhub\modules\user\models\Invite;
+use Yii;
+
+/**
+ * CleanupLog deletes older log records from log table
+ *
+ * @since 1.8
+ */
+
+class CleanupPendingRegistrations extends ActiveJob
+{
+    /**
+     * @var int seconds before delete old pending registrations messages
+     */
+    public $cleanupInterval = 60 * 60 * 24 * 90;
```

Comment:
@luke- question about the interval, is this related to Cron? Then wouldn't `60 * 60 * 24 * 7` be for the best? 
### See;
https://github.com/humhub/humhub/blob/a5da4c1b0b9eae3c38345f46abfa93ab8d030d42/protected/humhub/modules/admin/jobs/CleanupLog.php#L23-L26

Although I'm thinking something like `59 59 23 * * 7 *` would be better? As some Cron tasks don't calculate the same way :thinking:

Reply:
When commenting on this, I was wondering if this would work for all systems if it were related to cron jobs :thinking: